compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -1458,7 +1458,7 @@ def _on_message(
         message to AMQP service -- the responsibility to handle invalid messages
         is not with this class.
         """
-        msg_id: int = basic_deliver.delivery_tag
+        msg_id: int = basic_deliver.delivery_tag  # type: ignore
```

Comment:
Any idea why `# type: ignore` wasn't needed previously?

Reply:
Not sure?  I remember having to do it but I can try taking it out to see if latest mypy changes allow it.

Reply:
Still giving me the error, so I'm guessing newer mypy is more strict in this case.